### DISCUSSION TITLE
TF Plan - Is it possible to not display or mask backend config in PR comment?

### DISCUSSION THREAD

**Discussion by CharlesNRU** [dflook/terraform-github-actions#339] [Q&A] [answered]
When I run the terraform plan action, the comment added to the PR contains information about the backend configuration:
```
Terraform plan in .
With backend config: resource_group_name=<RG_NAME> storage_account_name=<SA_NAME> container_name=<CONTAINER_NAME> key=<KEY>
```

Is it possible to suppress or mask this information when using this action?

**Comment by dflook** [comment, marked as answer]
If you set the label input for the plan action, that will be shown in the PR comment instead of the backend config.

If you use the apply action, you will need to set the same label input there too.
